### PR TITLE
Feature/kitosudv 3939 odatav2 itsystem overview

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewInterfaceReadModel.cs [hunk 1]
```diff
@@ -1,9 +1,12 @@
-﻿namespace Core.DomainModel.ItSystemUsage.Read
+﻿using System;
+
+namespace Core.DomainModel.ItSystemUsage.Read
 {
     public class ItSystemUsageOverviewInterfaceReadModel
     {
         public int Id { get; set; }
 
+        public Guid InterfaceUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewInterfaceReadModel.cs:9]: remember to update mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewRoleAssignmentReadModel.cs [hunk 2]
```diff
@@ -1,9 +1,12 @@
-﻿namespace Core.DomainModel.ItSystemUsage.Read
+﻿using System;
+
+namespace Core.DomainModel.ItSystemUsage.Read
 {
     public class ItSystemUsageOverviewRoleAssignmentReadModel : IHasId
     {
         public int Id { get; set; }
 
+        public Guid RoleUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewRoleAssignmentReadModel.cs:9]: remember to update mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewUsedBySystemUsageReadModel.cs [hunk 3]
```diff
@@ -1,9 +1,12 @@
-﻿namespace Core.DomainModel.ItSystemUsage.Read
+﻿using System;
+
+namespace Core.DomainModel.ItSystemUsage.Read
 {
     public class ItSystemUsageOverviewUsedBySystemUsageReadModel : IItSystemUsageOverviewItSystemUsageReadModel
     {
         public int Id { get; set; }
 
+        public Guid ItSystemUsageUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewUsedBySystemUsageReadModel.cs:9]: remember to update mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewUsingSystemUsageReadModel.cs [hunk 4]
```diff
@@ -1,9 +1,12 @@
-﻿namespace Core.DomainModel.ItSystemUsage.Read
+﻿using System;
+
+namespace Core.DomainModel.ItSystemUsage.Read
 {
     public class ItSystemUsageOverviewUsingSystemUsageReadModel : IItSystemUsageOverviewItSystemUsageReadModel
     {
         public int Id { get; set; }
 
+        public Guid ItSystemUsageUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewUsingSystemUsageReadModel.cs:9]: remember to update mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs [hunk 5]
```diff
@@ -50,6 +50,7 @@ public ItSystemUsageOverviewReadModel()
         public Organization.Organization Organization { get; set; }
         public int Id { get; set; }
         public int SourceEntityId { get; set; }
+        public Guid SourceEntityUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs:53]: remember to add mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs [hunk 6]
```diff
@@ -59,14 +60,17 @@ public ItSystemUsageOverviewReadModel()
         public string Note { get; set; }
         public string ParentItSystemName { get; set; }
         public int? ParentItSystemId { get; set; }
+        public Guid? ParentItSystemUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs:63]: remember to add mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs [hunk 7]
```diff
@@ -59,14 +60,17 @@ public ItSystemUsageOverviewReadModel()
...
         public bool? ParentItSystemDisabled { get; set; }
         public string Version { get; set; }
         public string LocalCallName { get; set; }
         public string LocalSystemId { get; set; }
         public virtual ICollection<ItSystemUsageOverviewRoleAssignmentReadModel> RoleAssignments { get; set; }
         public string ItSystemUuid { get; set; }
+        public Guid? ResponsibleOrganizationUnitUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs:70]: remember mapping

File: Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs [hunk 8]
```diff
@@ -59,14 +60,17 @@ public ItSystemUsageOverviewReadModel()
...
         public int? ResponsibleOrganizationUnitId { get; set; }
         public string ResponsibleOrganizationUnitName { get; set; }
+        public Guid? ItSystemBusinessTypeUuid { get; set; }
```
Comment (mrjsawdk) [inline Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs:73]: add mapping

File: Tests.Integration.Presentation.Web/SystemUsage/ItSystemUsageOverviewReadModelsTest.cs [hunk 9]
```diff
@@ -53,6 +69,7 @@ public async Task Can_Query_And_Page_ReadModels()
             Assert.Equal(name3, page2.Single().SystemName);
         }
 
+        //TODO: Extend with the uuids
```
Comment (a-naskret) [inline Tests.Integration.Presentation.Web/SystemUsage/ItSystemUsageOverviewReadModelsTest.cs:72]: leftover todo

**Review by a-naskret — APPROVED** [review]

**Pull request merged**